gui/static/js/components/scenarios.js

```diff
@@ -163,13 +187,14 @@ export default class ScenariosItem extends React.Component {
             Button: {
                 marginLeft: '10px',
                 marginRight: '15px'
-            }
+            },
+
         }
         var _this = this;
         return(
             <div id = "scenariobox" className = "box">
                 <button onClick={this.saveScenario} style={styles.Button}>Save</button>
-                <button style={styles.Button}>Load</button>
+                <form><input type="file" style={styles.Button} accept=".txt" id="loadBtn" onChange={this.loadScenario}/></form>
```

Comment:
you could just it save and accept .json files (i think), it would make more sense than sticking with the old version that takes .txt

Reply:
The reason why I kept the file type as .txt is bc it's harder to edit a .json file. At least with my laptop, I cannot edit .json files directly.

Reply:
wait you can open it in notepad or sublime lol